ccd-definition/CaseField/CaseFieldLRspec.json

```diff
@@ -711,5 +711,42 @@
     "Label": " ",
     "FieldType": "TextArea",
     "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "responseClaimAdmitPartOfClaimWhenToPaySpecLabel",
+    "Label": "<h1 class=\"govuk-heading-l\">When does ${respondent1.partyName} want to pay the £${respondToAdmittedClaimOwingAmount}?</h1>",
```

Comment:
Please check which amount is to be shown here

Reply:
checked with Vartica, she thinks its the totalClaimAmount so defaulted to that. she's still checking with Alana